randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -177,17 +177,34 @@ Extra - asset_id: 4
   * Normal Door to The Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
       Any of the following:
-          # TODO: Check to see if possible with HBJ.
           Grapple Beam or Space Jump
-          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Horizontal Infinite Bomb Jump (Expert) and Can Lay Bombs
+          # It is possible to jump across by not using full jumps to avoid bonking on the celing.
           Movement (Intermediate)
           All of the following:
-              # From the sea floor
+              # From the Sea Floor
               Gravity Suit
               Any of the following:
                   Hi-Jump Boots or Wall Jumps (Beginner)
                   # IBJ
                   Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
+  > Door from Gravity Suit Room
+      Any of the following:
+          Space Jump
+          All of the following:
+              # Get over and WJ up
+              Grapple Beam or Movement (Intermediate)
+              Any of the following:
+                  Wall Jumps (Intermediate)
+                  Hi-Jump Boots and Wall Jumps (Beginner)
+          All of the following:
+              # From the Sea Floor
+              Gravity Suit
+              Any of the following:
+                  Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
+                  # Spark from the Sea Floor
+                  Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 51
+          Horizontal Infinite Bomb Jump (Expert) and Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
```

Comment:
This is still missing a comment

Reply:
A comment to say its IBJ + HBJ to the door?

Reply:
ye